android/src/main/java/com/android/simplSdk/SimplSdkModule.java

```diff
@@ -26,61 +26,96 @@ public SimplSdkModule(ReactApplicationContext reactContext) {
         super(reactContext);
     }
 
-    @Override
-    public String getName() {
-        return "SimplSdk";
+    @ReactMethod
+    public void isUserApproved(final ReadableMap map, final Callback successCallback, final Callback errorCallback){
+        final ReadableMap paramsMap = map.getMap("params");
+        try {
+            Simpl.init(getReactApplicationContext(), map.getString("merchantId"));
+            if (map.getBoolean("isSandbox"))
+                Simpl.getInstance().runInSandboxMode();
+
+            SimplUser user = new SimplUser(map.getString("email"), map.getString("phone_number"));
+            Simpl.getInstance().isUserApproved(user)
+                .addParam("transaction_amount_in_paisa", paramsMap.getString("amount_in_paisa"))
```

Comment:
You should pass all the params from the params map. Not just `transaciton_amount_in_paise`